codegen/templates/README.mustache

```diff
@@ -426,28 +502,55 @@ EX: Pagination do something per page
 let tasksApiInstance = new {{{moduleName}}}.TasksApi();
 let opts = {
     'project': "<YOUR_PROJECT_GID>",
-    'limit': 10
+    "limit": 5,
 };
 
-pageIndex = 1;
-
-tasksApiInstance.getTasks(opts).then(async (firstPage) => {
-    // Do something with the first <LIMIT> results. EX: print out results
-    console.log(`Page ${pageIndex}: ` + JSON.stringify(firstPage.data, null, 2));
+let pageIndex = 1;
 
-    // Get the next page
-    let nextPage = await firstPage.nextPage();
-    // Keep fetching for the next page until there are no more results
-    while(nextPage.data) {
-        // Do something with the next <LIMIT> results. EX: print out results
+tasksApiInstance.getTasks(opts).then(async (response) => {
+    let page = response;
+    while(true) {
+        // Do something with the page results
+        // EX: print the name of the tasks on that page
+        console.log(`Page ${pageIndex}: `);
+        page.data.forEach(task => {
+            console.log(`    ${task.name}`);
```

Comment:
Is that extra tab before task.name intended?

Reply:
Yes. The extra tab is intentional so that the print statement looks good.